src/components/side_panel/side_panel/side_panel.xml

```diff
@@ -3,21 +3,21 @@
     <div class="o-sidePanel">
       <div class="o-sidePanelHeader">
         <div class="o-sidePanelTitle" t-esc="getTitle()"/>
-        <div class="o-sidePanelClose" t-on-click="() => this.props.onCloseSidePanel()">✕</div>
+        <div class="o-sidePanelClose" t-on-click="close">✕</div>
       </div>
       <div class="o-sidePanelBody">
         <t
-          t-component="state.panel.Body"
-          t-props="props.panelProps"
-          onCloseSidePanel="props.onCloseSidePanel"
-          t-key="'Body_' + props.component"
+          t-component="panel.Body"
+          t-props="sidePanelStore.props"
+          onCloseSidePanel.bind="close"
+          t-key="'Body_' + sidePanelStore.component"
         />
       </div>
-      <div class="o-sidePanelFooter" t-if="state.panel.Footer">
+      <div class="o-sidePanelFooter" t-if="panel.Footer">
```

Comment:
Not related to the store, but that's the first time I realize we can define a Footer for the side panel 😅 . What's it for ? We never use it. And I have a hard time seeing where that'd be useful ? Probably some legacy stuff that we can remove, no?

Reply:
Has it ever been used ^^I think the idea was to have a common structure with the Odoo dialog but we never properly used it, for instance for the Cancel/ confirm buttons